<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255,255,255)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Edit Employee Table</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableView" name="tableView_RTM">
              <property name="minimumSize">
               <size>
                <width>700</width>
                <height>400</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>150</number>
              </property>
              <item>
               <widget class="QPushButton" name="reset_tab3_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(255,0,50,50,)</string>
                </property>
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="savechanges_tab3_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(255,0,50,50,)</string>
                </property>
                <property name="text">
                 <string>Save Changes</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="loadtable_tab3_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(255,0,50,50,)</string>
                </property>
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="add_employee_pushButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Add Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remove_row_pushButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Remove Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_row_pushButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Add Row</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>120</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="search_employee_comboBox">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Please select by</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Employee Id</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Employee name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Age</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gender</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Salary</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="search_employee_lineEdit">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="toolTip">
               <string>Enter Employee Id</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rg(0,0,50,10)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="search_employee_pushButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Search Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>48</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Import Table</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Export Table</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>User Table</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableView" name="tableView_user"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="add_user_pushButton">
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(255,0,50,50,)</string>
                </property>
                <property name="text">
                 <string>Add User</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove_user_pushButton_2">
                <property name="styleSheet">
                 <string notr="true">background-color:rgb(255,0,50,50,)</string>
                </property>
                <property name="text">
                 <string>Remove User</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>80</number>
            </property>
            <item>
             <widget class="QPushButton" name="reset_user_pushButton">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="savechanges_user_pushButton">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Save Changes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loadtable_user_pushButton">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255,0,50,50,)</string>
              </property>
              <property name="text">
               <string>Load table</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLog_Out"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="refresh_actionAbout"/>
    <addaction name="actionAdd_Employee"/>
    <addaction name="actionRemove_Employee"/>
    <addaction name="actionExport_Table"/>
    <addaction name="actionSave_Changes"/>
   </widget>
   <widget class="QMenu" name="menuVersion">
    <property name="title">
     <string>Version</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
   <addaction name="menuVersion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="refresh_actionAbout"/>
   <addaction name="actionAdd_Employee"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRemove_Employee"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExport_Table"/>
  </widget>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="refresh_actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/tool/rsrctool/refresh.png</normaloff>:/tool/rsrctool/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAdd_Employee">
   <property name="icon">
    <iconset>
     <normaloff>:/tool/rsrctool/write.png</normaloff>:/tool/rsrctool/write.png</iconset>
   </property>
   <property name="text">
    <string>Add Employee</string>
   </property>
  </action>
  <action name="actionRemove_Employee">
   <property name="icon">
    <iconset>
     <normaloff>:/tool/rsrctool/deleterow.png</normaloff>:/tool/rsrctool/deleterow.png</iconset>
   </property>
   <property name="text">
    <string>Remove Employee</string>
   </property>
  </action>
  <action name="actionExport_Table">
   <property name="icon">
    <iconset>
     <normaloff>:/tool/rsrctool/show table.png</normaloff>:/tool/rsrctool/show table.png</iconset>
   </property>
   <property name="text">
    <string>Export Table</string>
   </property>
  </action>
  <action name="actionSave_Changes">
   <property name="text">
    <string>Save Changes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
